examples/README.md

```diff
@@ -0,0 +1,177 @@
+# Create a new op in C++, compile it to .so library and load it in Python

```

Comment:
Do we need to move this new created "custom_converter" folder in some other directories?  like "notebooks" ? Since "examples" directory is new created for this task.

Reply:
I think the folder structure should be something like 

```
//examples 
    - custom_converters/
        - README.md
        - elu_model.py (your elu_converter_test.py)
        - elu_converter/
            - setup.py
            - elu_converter/
                - csrc/
                    - elu_converter.cpp 
```